README.adoc

```diff
@@ -343,6 +344,14 @@ The following table documents the patches we've made to PhantomJS and tracks the
 |{icon-exclamation-dim} todo
 |===
 
+=== WebSlides
```

Comment:
Funny, that's where I had it first and then it seemed a bit too prominent... (since it's only for that one framework). But it does make sense, that's probably where you'd look for it, will move it.